tests/pos/nullAsInstanceOf.scala

```diff
@@ -1,11 +0,0 @@
-object Test {
```

Comment:
for future reference, you can just write `extends dotty.runtime.LegacyApp` and body will be executed.